KomaMRICore/src/datatypes/Phantom.jl

```diff
@@ -136,92 +160,6 @@ end
 	)
 end
 
-# Movement related commands
-# StartAt(s::Phantom,t0::Float64) = Phantom(s.name,s.x,s.y,s.ρ,s.T2,s.Δw,s.Dλ1,s.Dλ2,s.Dθ,(x,y,t)->s.ux(x,y,t.+t0),(x,y,t)->s.uy(x,y,t.+t0))
-# FreezeAt(s::Phantom,t0::Float64) = Phantom(s.name*"STILL",s.x.+s.ux(s.x,s.y,t0),s.y.+s.uy(s.x,s.y,t0),s.ρ,s.T2,s.Δw,s.Dλ1,s.Dλ2,s.Dθ,(x,y,t)->0,(x,y,t)->0)
-
-#TODO: jaw-pitch-roll, expand, contract, functions
-
-# Getting maps
-# get_DxDy2D(obj::Phantom) = begin
-# 	P(i) = rotz(obj.Dθ[i])[1:2,1:2];
-# 	D(i) = [obj.Dλ1[i] 0;0 obj.Dλ2[i]]
-# 	nx = [1;0]; ny = [0;1]
-# 	Dx = [nx'*P(i)'*D(i)*P(i)*nx for i=1:prod(size(obj.Dλ1))]
-# 	Dy = [ny'*P(i)'*D(i)*P(i)*ny for i=1:prod(size(obj.Dλ1))]
-# 	Dx, Dy
-# end
-
-"""
-    phantom = heart_phantom(α=1, β=1, γ=1, fat_bool::Bool=false)
-
-Heart-like LV phantom. The variable `α` is for streching, `β` for contraction, and `γ` for
-rotation.
-
-# Arguments
-- `α`: (`::Real`, `=1`) streching parameter
-- `β`: (`::Real`, `=1`) contraction parameter
-- `γ`: (`::Real`, `=1`) rotation parameter
-- `fat_bool`: (`::Bool`, `=false`) fat boolean parameter
-
-# Returns
-- `phantom`: (`::Phantom`) Heart-like LV phantom struct
-"""
-heart_phantom(α=1, β=1, γ=1, fat_bool::Bool=false) = begin
```

Comment:
Instead of removing this one, I would try to make it work again.

Reply:
I was doing some tests, apparently the heart phantom its modeled with a circular shape and has movement. The way it's implemented the spins (or bulk-magnetizations) are arranged in a 2D matrix, however it should be a Vector of spins. I think we should definitely make it work later.